package.json

```diff
@@ -41,8 +41,7 @@
   },
   "lint-staged": {
     "*.{js,css,scss,json,md,ts,tsx,html,yml}": [
-      "prettier --write",
-      "git add"
+      "prettier --write"
```

Comment:
I removed that due to a warning of the latest `lint-staged` and that caused the package lock update.